app/views/provider_interface/organisations/_permissions_list.html.erb

```diff
@@ -0,0 +1,12 @@
+<ul class="govuk-list">
+  <% if permission.view_safeguarding_information %>
+    <li>
+      <%= render 'check_icon' %>
+      access safeguarding information
+    </li>
+  <% end %>
+  <li>
```

Comment:
Do we need an org-level 'Make decisions` entry?

Reply:
@mnacos not sure if this is an org level permission, I think if it is, we should do it in a separate PR to keep this moving as we'll need to amend `ProviderRelationshipPermissions` with a suitable migration etc.

Reply:
As an organisation, I'd want to be able to grant/withdraw an org-level 'Make decisions' but I guess we should double-check. Like you say, this can be done as a separate PR.